packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -185,6 +185,7 @@ const metadata: Meta<TableArgs> = {
             <${tableColumnTextTag}
                 column-id="favorite-color-column"
                 field-name="favoriteColor" placeholder="no value"
+                fractional-width=".5"
```

Comment:
Just a note that currently there's no Storybook docs dealing with column widths, although we do configure column widths here.
I saw a comment on @mollykreis 's sorting PR about creating a new `table-column.stories.ts` for having column-related docs. I didn't do that yet / in this PR though.

Reply:
Chatted with @jattasNI about this, and we will follow up his PR that will introduce column-specific storybooks, with one that deals with column widths.

Reply:
#1105 for reference